build-parent/pom.xml

```diff
@@ -26,7 +26,7 @@
         <version.buildnumber.plugin>3.1.0</version.buildnumber.plugin>
         <version.cucumber.reporting.plugin>5.7.5</version.cucumber.reporting.plugin>
         <version.failsafe.plugin>${version.maven.surefire.plugin}</version.failsafe.plugin>
-        <version.fermenter>2.10.2</version.fermenter>
+        <version.fermenter>2.10.3-SNAPSHOT</version.fermenter>
```

Comment:
Q: Is this worth cutting a fermenter release?

Reply:
This is the first feature added since the previous release, as far as I know there aren't any plans for an immediate release for just this addition. That may be a question for Ryan if you want to raise the possibility of cutting one

Reply:
I think going forward we probably want to either cut the release or keep snapshot updates on a feature branch and release before merging back to dev.  Unfortunately I didn't communicate this very clearly with @ahartwellCpointe offline and ended up implying we should merge this snapshot version into dev.  We're a bit off still from trying to release 1.7 so as long as we focus on getting the follow-on work for this in and cut the fermenter release for that PR we should be fine.

Reply:
Actually. Because Maven Central doesn't allow snapshot publishing anymore, we'll need to roll this back as there's no public jar matching this version